common/views.py

```diff
@@ -909,3 +909,42 @@ def post(self, request):
         response['refresh_token'] = str(token)
         response['user_id'] = user.id
         return Response(response)
+
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
I don't know how the logger works really, I think we can remove this line and `import logging` to have a cleaner code.